src/content/guides/creating-clusters-via-crs-on-aws/index.md

```diff
@@ -0,0 +1,169 @@
++++
+title = "Creating tenant clusters on AWS via Control Plane Kubernetes API"
+description = "This guide will walk you through the process of tenant cluster creation via Control Plane Kubernetes."
+date = "2020-04-01"
+type = "page"
+weight = 100
+tags = ["tutorial"]
++++
+# Creating tenant clusters via the Control Plane Kubernetes API
+
+This guide will show you how to create tenant clusters by creating and applying CRs directly to the control plane.
+Previously you might have used our REST API to create clusters, however, Giant Swarm is replacing its own REST API for cluster management with the [Control Plane](https://docs.giantswarm.io/basics/aws-architecture/#giant-swarm-control-plane) Kubernetes API based on the upstream [Cluster API](https://cluster-api.sigs.k8s.io/).
+Following this strategy, the Giant Swarm API is going to be deprecated in the near feature. You can find the related roadmap issue [here](https://github.com/giantswarm/roadmap/issues/90).
+
+
+## How does cluster creation work now?
+
+Starting from version 10.0.0, Giant Swarm introduced a feature to create multiple [node pools](https://docs.giantswarm.io/basics/nodepools/) on AWS.
```

Comment:
might make more sense with 'from version 10.0.0 on AWS, Giant Swarm introduced...` - otherwise it's easy to miss that this is not true for say, 10.0.0 on Azure